langchain/src/vectorstores/milvus.ts

```diff
@@ -421,7 +440,7 @@ export class Milvus extends VectorStore {
       if (field.is_primary_key) {
         this.primaryField = field.name;
       }
-      const dtype = DataTypeMap[field.data_type.toLowerCase()];
+      const dtype = DataTypeMap[field.data_type];
```

Comment:
In new sdk, data_type is keyof typeof DataType. So we don't need to lowerCase here, otherwise ts will throw error.